contacts-app/src/components/ContactForm/ContactForm.jsx

```diff
@@ -81,7 +134,41 @@ class ContactForm extends React.Component {
   }
 }
 
+ContactForm.propTypes = {
+  addNewContact: PropTypes.func,
+  contactId: PropTypes.number
+};
+
 export default ContactForm;
 
-const defaultImage =
-  "https://avataaars.io/?avatarStyle=Transparent&topType=ShortHairShaggyMullet&accessoriesType=Prescription02&hairColor=Brown&facialHairType=Blank&clotheType=GraphicShirt&clotheColor=White&graphicType=Resist&eyeType=Wink&eyebrowType=Default&mouthType=Smile&skinColor=Light";
+const defaultImage = avatar1;
```

Comment:
I would maybe move this to the top of the file before the class so it doesnt get lost.